packages/react/src/drawer/docs/overview.mdx

```diff
@@ -347,3 +347,64 @@ By default, Drawer’s overlay defaults to 80% opacity, however, by adding `mute
 	);
 };
 ```
+
+## Focus custom element on close
+
+When the drawer closes, focus is returned to the element that triggered its opening. You can override this behaviour by passing a custom element to the `elementToFocusOnClose` prop. This is useful when an action performed in the drawer causes an error and focus should be returned to an alert on close.
```

Comment:
OK how about this?
```suggestion
When a drawer closes, focus is returned to the element that opened it. You can override this behaviour, by passing an element ref to the `elementToFocusOnClose` prop. This can be used in situations such as when an action performed in the drawer causes an error on the main page, thus requiring the user’s focus to be sent to the alert instead.
```